control.go

```diff
@@ -59,6 +61,13 @@ func StartControlServer(logger log.Logger, manager device.Manager, deviceGate de
 		apiHandler = r.PathPrefix(fmt.Sprintf("%s/%s", baseURI, version)).Subrouter()
 	)
 
+	otelMuxOptions := []otelmux.Option{
+		otelmux.WithPropagators(tracing.Propagator),
+		otelmux.WithTracerProvider(tracing.TracerProvider),
+	}
+
+	apiHandler.Use(otelmux.Middleware("mainSpan", otelMuxOptions...), candlelight.EchoFirstTraceNodeInfo(tracing.Propagator))
```

Comment:
```suggestion
	r.Use(otelmux.Middleware("primary", otelMuxOptions...), candlelight.EchoFirstTraceNodeInfo(tracing.Propagator))
```
I'm not sure if there would be a behavior difference but suggesting this so we keep it consistent with our other services.

Reply:
Since there are multiple servers we're instrumenting, it'll be better to use those names to differentiate them easily.